### PR TITLE
fix gc activity got recorded in request seconds

### PR DESCRIPTION
**Pull request by zhexuany**
@coocood @disksing 
PTAL

This PR aims to make our monitor panel more readable.

### CODE REVIEW

File: store/tikv/client.go [hunk 1]
```diff
@@ -221,7 +221,10 @@ func (c *rpcClient) SendRequest(ctx context.Context, addr string, req *tikvrpc.R
 	reqType := req.Type.String()
 	storeID := strconv.FormatUint(req.Context.GetPeer().GetStoreId(), 10)
 	defer func() {
-		metrics.TiKVSendReqHistogram.WithLabelValues(reqType, storeID).Observe(time.Since(start).Seconds())
+		// skip taking samples when request type is gc.
+		if req.GC != nil {
```
Comment (coocood) [inline store/tikv/client.go]: Do you mean `req.GC == nil`?

**Comment by shenli** [comment]
@zhexuany Could you just ignore certain data by modifying the formula on Grafana?

**Comment by shenli** [comment]
I thought this is already done. See: https://github.com/pingcap/tidb-ansible/blob/master/scripts/tidb.json#L3418

**Comment by zhexuany** [comment]
@shenli Yes, but the lira issue is not updated. I thought it is still remaining open. Closing for good.

**Pull request closed without merging**